src/poggit/release/details/ReleaseDetailsModule.php

```diff
@@ -740,8 +740,7 @@ class="remark"><?= $this->reqr["isRequire"][$key] === 1 ? "Requirement" : "Enhan
                   </div>
                 <?php } ?>
                 <?php if(count($this->authors) > 0) { ?>
-                  <div class="plugin-info-wrapper" id="release-authors"
-                       data-owner="<?= $this->release["author"] ?>">
```

Comment:
Note this removes the 'Owner' section in plugins producer list when on the specific page eg https://poggit.pmmp.io/p/DiscordBot/2.0.3 is what it will look like but without the owner list when others are listed now. (eg https://poggit.pmmp.io/p/JukeBox)